hiveadmission-certs/server-key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIOtr50NtrbrJOe2czj8y/aQQbi1vKOKNDomTFmzHJLBWoAoGCCqGSM49
+AwEHoUQDQgAEv3avKHabVWLOun5GazbyTGEhAUPvDmaAa1QZZHypMbOyHVOGG8HW
+BRCeHJZyoMOEaWDW7PqBxgSAe/ZImOnfKQ==
+-----END EC PRIVATE KEY-----
```

Comment:
Directory (3 files) mistakenly added to commit.

Reply:
oops